site/content/integrate/admin-guide/archive/README.md

```diff
@@ -0,0 +1 @@
+
```

Comment:
I've added that for the files I transferred over and kept in `.md` format in case the new structure didn't work out. Happy to delete it when I've confirmed it's all displaying as needed.